js/main.js

```diff
@@ -17,5 +26,125 @@ const navDisappear = () => {
     });
   }
 };
+
+const popUp1 = () => {
+  projectButton.addEventListener('click', () => {
+    modal.classList.add('modal-appear');
+  });
+};
+
+const projectCollection = {
+  project1: {
+    heading: 'Multi Post Stories',
+    tag1: 'html',
+    tag3: 'Bootstrap',
+    tag2: 'Ruby on rails',
+    imageLink: '<div class="modalImage"><img class="noTenImg" src="images/tenBig.png" alt="Big Number 10"></div>',
+    paragraph: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+  },
+  project2: {
+    heading: 'Empty',
+    tag1: 'Empty',
+    tag3: 'Empty',
+    tag2: 'Empty',
+    imageLink: '<div class="modalImage"><img class="noTenImg" src="imagestenBig.png" alt="Big Number 10"></div>',
+    paragraph: 'Empty',
+  },
```

Comment:
- [ ] It is a requirement to store your project's data in an array of objects. `const projectCollection = [ {..}, {...}, ... ]`